git-cliff/src/lib.rs

```diff
@@ -155,31 +155,36 @@ pub fn run(mut args: Opt) -> Result<()> {
 		}
 	} else if args.latest || args.current {
 		if tags.len() < 2 {
-			return Err(Error::ChangelogError(String::from(
-				"Not enough tags exist for processing the latest/current tag",
-			)));
-		}
-		let mut tag_index = tags.len() - 2;
-		if args.current {
-			if let Some(current_tag_index) =
-				repository.current_tag().as_ref().and_then(|tag| {
-					tags.iter()
-						.enumerate()
-						.find(|(_, (_, v))| v == &tag)
-						.map(|(i, _)| i)
-				}) {
-				tag_index = current_tag_index - 1;
-			} else {
-				return Err(Error::ChangelogError(String::from(
-					"No tag exists for the current commit",
-				)));
+			let commits = repository.commits(None, None, None).unwrap();
```

Comment:
I assume you didn't specify `include_path` and `exclude_path` arguments because this list is only used for getting a commit range, right? They are used in the later `commits` call anyways.

Can you use `?` instead of `unwrap()`?

Reply:
> I assume you didn't specify include_path and exclude_path arguments because this list is only used for getting a commit range, right?

Yes, I saw this line. But I'm not sure why I replaced `?` with `unrwap()`. :cry: 
https://github.com/orhun/git-cliff/blob/ff1d981fd9a5dba26422f56582e06b3b463eb8a3/git-cliff-core/src/repo.rs#L181

> Can you use ? instead of unwrap()?

Sure, fixed in 093e9a0dc3cc0d6ab7b12ea576b365cf02f1c32f.
BTW, could you show me some references for me? I'm quite a newbie in Rust and couldn't find a good page.

Reply:
You don't seem that new to me :) Either way, here you go:

- https://doc.rust-lang.org/rust-by-example/error.html
- https://github.com/rust-lang/rustlings
- https://doc.rust-lang.org/book/ (obviously)